### DISCUSSION TITLE
Networking support is coming to MiniDexed

### DISCUSSION THREAD

**Discussion by probonopd** [probonopd/MiniDexed#759] [Announcements]
![image](https://github.com/user-attachments/assets/220244fd-ed12-42ad-8269-e299dbba698a)

Learn all about it here:

https://github.com/probonopd/MiniDexed/wiki/Networking

We'd appreciate your testing and [feedback](https://github.com/probonopd/MiniDexed/pull/747).

A big "thank you" to

* [dwhinham/mt32-pi](https://github.com/dwhinham/mt32-pi) for creating networking support for circle
* [omersiar](https://github.com/omersiar) for porting networking support to MiniDexed

**Comment by Lonolo** [comment]
A very big "Thank You"!

Works on Zero2 - receives from a Mac using Dexed, Ableton Live, even 'ole Voyetra SPG running on DOSbox.
![IMG_3806](https://github.com/user-attachments/assets/66160bd8-0175-406d-9817-8ba085d00607)

**Reply by omersiar** [reply]
How do you send MIDI from DosBox?

**Reply by Lonolo** [reply]
From DosBox straight to, and through Dexed.

Using a Mac, I have SPG connected to Dexed with settings below, and Dexed MIDI output uses the Network Session as a thru from SPG. On all channels - not just what is set in Dexed. I set the TG's in MiniDexed to MIDI channels 1-8 respectively and sequence them from SPG. It worked with hardware MIDI and now, much thanks to you through WIFI.

DOSbox preferences:
mpu401= intelligent
mididevice= default
sbtype=sb1
sbbase=220
irq=7
dma=1
hdma=5

Unfortunately, DOSbox does not support MIDI input as far as I know.
